sam.c

```diff
@@ -3710,11 +3715,25 @@ static int fastq_parse1(htsFile *fp, bam1_t *b) {
     }
 
     // Name
+
+    // Reverse the NCBI strangeness of putting the run_name.number before
+    // the read name.
+    i = 0;
+    char *name = x->name.s+1;
+    if (x->ncbi_names) {
+        char *cp = strchr(x->name.s, ' ');
+        if (cp) {
+            *cp = '@';
+            i = cp - x->name.s;
+            name = cp+1;
+        }
+    }
+
```

Comment:
A minor point, but it might be good to move this after the `x->nprefix` test below.  As it is, it will "fix" files with lines that don't have the correct prefix, but do start with a leading space, which may or may not be a good thing...

Also, how general do we want this.  Should we be looking for multiple spaces or tabs?

Reply:
I could rename it FASTQ_OPT_SRA.  I don't really like second name as it's long winded and like it or not this *is* an SRA specific format.  ENA only use it because SRA use it.

Reply:
Actually I'll take that back, I see what you mean.  Maybe ERA fixed SRA format by still included the data.  I'm not really sure.  SRA came first though so I mentally peg this as their error.  I'll ponder.